Bottom: 3fadfa6ab4dbfd8c9d3fadf64b1e48c29b17fb74
Top:    2142b4518c0c44d79db74deaf3ae6191c84844ac
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-08-16 09:17:44 -0500

Enable file-based secrets provider


---
diff --git a/docker/buildbot-master/buildbot/master.cfg b/docker/buildbot-master/buildbot/master.cfg
index fb54cb31..52c07e46 100644
--- a/docker/buildbot-master/buildbot/master.cfg
+++ b/docker/buildbot-master/buildbot/master.cfg
@@ -5,7 +5,7 @@
 # 'master.cfg' in your buildmaster's base directory.
 
 import os
-from buildbot.plugins import steps, util
+from buildbot.plugins import secrets, steps, util
 
 os.environ['LONG_RUN_TIMEOUT'] = '14400'
 
@@ -22,6 +22,9 @@ c['change_source'] = []
 # --master option)
 c['protocols'] = {"pb": {"port": int(os.getenv('BUILDBOT_WORKER_PORT', 9989))}}
 
+# SECRETS MANAGEMENT
+c['secretsProviders'] = [secrets.SecretInAFile(dirname="/run/secrets")]
+
 # STATUS TARGETS
 
 # 'status' is a list of Status Targets. The results of each build will be
